source/clear-linux/guides/maintenance/ister.rst

```diff
@@ -0,0 +1,89 @@
+.. _ister:
+
+ister.py
+########
+
+The `ister.py tool`_ is a template based installer used by |CL-ATTR| to produce
+images for each release. The same ister tool is available for use in |CL| to
+create custom images based on an upstream image.
+
+Description
+***********
+
+|CL| is a rolling release and produces on average 10 releases per week using the
+ister tool. With each release we produce multiple
+`image types for different environments`_  and use cases such as installers,
+Hyper-V, KVM, or VMWare.
+
+Each image has a JSON configuration file used by ister to generate the image.
+These JSON configuration files describe the image type, partitions, version,
+and which bundles will be preinstalled by default with the image. For each image
+type we produce, the corresponding JSON configuration file for the image is also
+published.
+
+The :ref:`mixer<mixer>` tool also uses ister to build images for your custom
+mix. Like upstream images, a JSON configuration file is defined for the image,
+which ister uses to generate the image. Refer to the :ref:`mixer<mixer>` guide
+for instruction on using ister to build an image for a custom mix.
+
+Example: Recreate an upstream image
+***********************************
+
+The published configuration files for upstream images may be used to recreate an
+image, for example when you want to:
+
+* Use an older version of |CL| and the image is no longer available (only after
+  March 2017).
+* Customize the partitions of an image.
+* Customize the bundles preinstalled in an image.
+* Run your own post installation script.
+
+Prerequisites
+=============
+
+#. |CL| installed.

```

Comment:
Please remove periods for bullet fragments.  Looked but couldn't find a BKM on this. Please advise if there is one.

Reply:
There is - https://clearlinux.org/documentation/clear-linux/reference/collaboration/structure-formatting#lists-and-instructions